c31-bywheat/by-wheat-nutshell-react
# Cm edit button

# Description

Edit button and edit functionality now works as inteded.

Applies to issue # 5

## Type of change

Please delete options that are not relevant.



# Testing Instructions

Clone branch and run servers.
Make sure to have data in your messages.
Messages should appear and when edit button is clicked will allow you to edit existing message.



# Checklist:
